### PR TITLE
Add summit sponsor collins

### PR DESCRIPTION
**Pull request by bbrcknl**

### CODE REVIEW

File: _news-items/2024-08-16-collins.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+# Copyright 2024 seL4 Project a Series of LF Projects, LLC.
+# SPDX-License-Identifier: CC-BY-SA-4.0
+title: Welcome Collins, Bronze sponsor of the seL4 Summit 2024
+anchor: sponsor24-collins
```
Comment (lsf37) [inline _news-items/2024-08-16-collins.html:5]: can leave this one out for new items

File: _news-items/2024-08-16-collins.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <img src="../images/summit/sel4_summit_2024.svg" style="width: 15%; float:right; padding-bottom:5px"
+      alt="seL4 summit">
+  </a>
+  It was a prime contractor in the DARPA HACMS
+  program, which demonstrated the seL4-based incremental cyber retrofit of
+  autonomous military vehicles. This was a major milestone in the growth of seL4,
+  demonstrating that it protects against cyber attacks on real systems in
+  operation. The same team also led the follow-on DARPA CASE program, aiming at
+  designed-in cyber-resiliency, including the seL4-based framework for verified
+  initialisation and configuration of systems architectures.`
```
Comment (june-andronick) [inline _news-items/2024-08-16-collins.html:30]: There's an extra back tick at the end of the line

Comment (june-andronick) [inline _news-items/2024-08-16-collins.html:30]: Add: "Collins Aerospace is now leading the INSPECTA team as part of DARPA's PROVERS program, which aimsat developing formal methods tools to guide software engineers through designing proof-friendly software systems and reduce the proof repair workload."

File: _news-items/2024-08-16-collins.html (hunk 1 above)
Comment (bbrcknl) [reply _news-items/2024-08-16-collins.html:5]: But we need the title and anchor, don't we?

Comment (bbrcknl) [reply _news-items/2024-08-16-collins.html:3]: Removing these lines also results in a licence error. Pls see below.

Comment (lsf37) [reply _news-items/2024-08-16-collins.html:3]: I meant just the `anchor` line, the rest we do need.

Comment (lsf37) [reply _news-items/2024-08-16-collins.html:3]: (Should have realised that GitHub will show more lines of context -- sorry for the confusion)

Comment (bbrcknl) [reply _news-items/2024-08-16-collins.html:3]: Oh righto. I'll fix it now. Thanks for letting me know!

**Pull request merged**